nuclia/sdk/search.py

```diff
@@ -60,6 +62,13 @@ def _auth(self) -> NucliaAuth:
 
     @kb
     @pretty
+    @backoff.on_exception(
+        backoff.expo,
+        exceptions.UnknownError,
```

Comment:
This exception is raised for any 5xx error, so we'll be retrying all of them by default.

I'm not against it, but just to have it in mind. I think it would be better to retry specific status codes that we know will be resolved.

What intermittent error are you trying to fix?